experiments/optimal-core-allocation/README.md

```diff
@@ -2,32 +2,32 @@
 
 ## Rationale
 
-This experiment allows you to find the optimal core and thread workers configuration that delivers the best performance 
-in terms of throughput and guaranteeing latency for sensitive workloads.
+This experiment allows you to find the optimal core and worker threads configuration that delivers the best performance 
+in terms of throughput and guaranteeing response time for latency sensitive workloads.
 
-This example experiment uses memcached as the latency sensitive workload and tries to find optimal number of memcached worker threads (typically set with  the `-t` flag for `memcached`).
+The experiment uses memcached as the latency sensitive workload and tries to find optimal number of memcached worker threads (typically set with  the `-t` flag for `memcached`).
 
-#### Environment prerequisites
+### Environment prerequisites
```

Comment:
Should we write that all those things are available in "prepare_experiment_environment.sh"?